ckan/plugins/toolkit.py

```diff
@@ -99,7 +100,7 @@
     "mail_recipient", "mail_user",
     "render_snippet", "add_template_directory", "add_public_directory",
     "add_resource", "add_ckan_admin_tab",
-    "check_ckan_version", "requires_ckan_version", "get_endpoint",
+    "check_ckan_version", "requires_ckan_version", "get_endpoint", "csrf_input"
```

Comment:
No need to add this to the toolkit, it's already accessible to templates (and to python code) via the helper.